src/components/SearchResults.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "../styles/search-results.css";
+
+const SearchResults = ({ results }) => {
+  if (!results) {
+    return <p></p>
+  } else if (!results.length) {
+    return <p className="results-message">No images found</p>
+  } else {
+    return(
+      <div className="search-results">
+        {results.map((result) => (
+          <img 
+            key={result}
+            className="search-results__image"
+            data-testid="search-results__image"
+            src={result}
+            alt="space"
```

Comment:
This alt tag isnt all that helpful. Their purpose is to describe what an image is for people using a screen reader, in this case all the images will have this generic label that doesnt help describe anything really. A good next step for future development would be to see if you can get the alt tag for each image from the API and attach it to its image